services/worker/src/worker/main.py

```diff
@@ -32,9 +39,15 @@ async def every(
         await asyncio.sleep(seconds)
 
 
+class WorkerCrashedError(CustomError):
+    def __init__(self, message: str, cause: Optional[BaseException] = None):
+        super().__init__(message, HTTPStatus.NOT_IMPLEMENTED, "WorkerCrashedError", cause, True)
```

Comment:
Not sure if we should disclose the error to the user here - maybe I should set it to False
```suggestion
        super().__init__(message, HTTPStatus.NOT_IMPLEMENTED, "WorkerCrashedError", cause, False)
```

Reply:
Agree, since it is an internal error that we should handle it is better to False